crates/bevy_winit/src/lib.rs

```diff
@@ -437,6 +438,16 @@ pub fn winit_runner(mut app: App) {
                     return;
                 };
 
+                // Allow AccessKit to filter `WindowEvent`s before they reach
+                // the engine.
+                if let Some(adapter) = access_kit_adapters.get(&window_entity) {
+                    if let Some(window) = winit_windows.get_window(window_entity) {
+                        if !adapter.on_event(window, &event) {
+                            return;
+                        }
```

Comment:
```suggestion
                        // AccessKit might later need to filter events based on this
                        // See https://github.com/bevyengine/bevy/pull/10239#issuecomment-1775572176
                        adapter.on_event(window, &event);
```

Reply:
I feel weird about this because it seems a bit arbitrary. There's already a comment explaining what's going on. It isn't scoped to the specific API call being set up, sure, but I don't think it needs to be called out twice.

That said, I wouldn't mind seeing documentation on this from AccessKit's end, particularly as there isn't any ATM. Then you could just F12 to the reference and have the most up-to-date explanation for why it's needed right there. I just don't want to set a precedent for having to justify individual AccessKit API calls.